firebase-config/src/main/java/com/google/firebase/remoteconfig/RemoteConfigComponent.java

```diff
@@ -299,4 +324,43 @@ private static boolean isAbtSupported(FirebaseApp firebaseApp, String namespace)
   private static boolean isPrimaryApp(FirebaseApp firebaseApp) {
     return firebaseApp.getName().equals(FirebaseApp.DEFAULT_APP_NAME);
   }
+
+  private synchronized void notifyRCInstances(boolean background) {
+    for (FirebaseRemoteConfig frc : frcNamespaceInstancesBackground.values()) {
+      if (!background) {
+        // Add dummy listener and remove to trigger http stream flow.
+        ConfigUpdateListenerRegistration registration =
+            frc.addOnConfigUpdateListener(
+                new ConfigUpdateListener() {
```

Comment:
Nit. You can optionally create a constant named `EMPTY_CONFIG_LISTENER` or `NOOP_CONFIG_LISTENER`, set it to be this object, and assign it here. That would make this snippet easier to read.